File: nom-api/Nom.Orch/Models/Household/HouseholdMemberResponseModel.cs
// File: Nom.Orch/Models/Household/HouseholdMemberResponseModel.cs

namespace Nom.Orch.Models.Household
{
    public class HouseholdMemberResponseModel
    {
        public long Id { get; set; }
        public long HouseholdId { get; set; }
        public long PersonId { get; set; }
        public string PersonName { get; set; } = string.Empty;
        public string? PersonEmail { get; set; }
        public string Role { get; set; } = string.Empty;
        public DateTime JoinedDate { get; set; }
        public bool IsActive { get; set; } = true;
        public bool HasProfile { get; set; }
        public bool HasRestrictions { get; set; }
    }
} 

File: nom-ui/src/app/core/models/household-member-response.model.ts
export interface HouseholdMemberResponseModel {
  id: number;
  householdId: number;
  personId: number;
  personName: string;
  personEmail: string | null;
  role: string;
  joinedDate: string;
  isActive: boolean;
  hasProfile: boolean;
  hasRestrictions: boolean;
}
